src/foam/nanos/cron/Frequency.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.ENUM({
+  package: 'foam.nanos.cron',
+  name: 'Frequency',
```

Comment:
This needs be renamed to ScheduleFrequency to distinguish it from a cron frequency which also includes seconds, minutes, hours. 
And why are avoiding seconds, minutes, hours?  
Also, foam.time.TimeUnit has conversionFactorMs so we can convert between them.  If we don't use that here, that same setup here will be useful, also for converting the enum to a millisecond equivalent in general.

Reply:
@jlhughes In the previous pr[(NP-6537)](https://github.com/kgrgreer/foam3/pull/1426), we used foam.time.TimeUnit for the "frequency" property in SimpleIntervalSchedule.js. so we don't need this anymore.
Also, in the pr(NP-6537) I added all features from this pr to the previous pr. Do we still need this pr though?

Reply:
If the SimpleIntervalSchedule is in the other PR then yes, you can close this one.